plugins/catalog-react/src/hooks/useEntityPermission.ts

```diff
@@ -15,22 +15,24 @@
  */
 
 import { stringifyEntityRef } from '@backstage/catalog-model';
-import { Permission } from '@backstage/plugin-permission-common';
+import { ResourcePermission } from '@backstage/plugin-permission-common';
 import { usePermission } from '@backstage/plugin-permission-react';
 import { useEntity } from './useEntity';
 
 /**
  * A thin wrapper around the
  * {@link @backstage/plugin-permission-react#usePermission} hook which uses the
  * current entity in context to make an authorization request for the given
- * permission.
+ * {@link @backstage/plugin-catalog-common#CatalogEntityPermission}.
  *
  * Note: this hook blocks the permission request until the entity has loaded in
  * context. If you have the entityRef and need concurrent requests, use the
  * `usePermission` hook directly.
  * @alpha
  */
-export function useEntityPermission(permission: Permission): {
+export function useEntityPermission(
+  permission: ResourcePermission<'catalog-entity'>,
```

Comment:
Could use the `CatalogEntityPermission` convenience type here

Reply:
Totally! This is actually the reason I added the convenience type in the first place 😅 . Unfortunately it's not currently possible to re-export any types including alpha types from other packages - see [this thread](https://github.com/backstage/backstage/pull/10066#discussion_r821893489), and https://github.com/backstage/backstage/pull/10128 which contains a proof-of-concept for fixing the issue.